extra_tests/snippets/stdlib_io.py

```
@@ -1,5 +1,5 @@
 import os
-from io import BufferedReader, BytesIO, FileIO, RawIOBase, StringIO
+from io import BufferedReader, BytesIO, FileIO, RawIOBase, StringIO, TextIOWrapper
```

Flake8 (7.3.0)

[error] 'io.BytesIO' imported but unused

(F401)